### PR TITLE
Modify the cf template to support login branding and home redirect properties

### PR DESCRIPTION
**Pull request by Kaixiang**
Hi:

Cloudops would like to be able specify the branding and home redirect properties in our stub file, 
Here is the PR for it, could you help review it. 

With many thanks.

### CODE REVIEW

**Comment by Amit-PivotalLabs** [comment]
I think the template change is fine, but I'd rather not have that stuff added to the `spec/fixtures/aws/cf-stub.yml`.  Those stubs are used to document how to get started with CF, and having that stuff is unnecessarily confusing for new users.  It's also inconsistent with all the other IaaS stubs.  Would you mind removing that part from the AWS stub and making the appropriate change to the AWS manifest fixture?

**Comment by Amit-PivotalLabs** [comment]
Hey @Kaixiang 

Could you remove the stuff from the AWS stub altogether?  And then squash everything into one commit please?

Thanks,
Amit

**Comment by Kaixiang** [comment]
@Amit-PivotalLabs : squashed into one commit.

**Comment by Amit-PivotalLabs** [comment]
Thanks @Kaixiang 

I'm still requesting not adding the changes to the AWS stub.

File: spec/fixtures/aws/cf-stub.yml [hunk 1]
```diff
@@ -127,6 +127,9 @@ properties:
         name: uaadb
     address: UAADB_ADDRESS
     port: UAADB_PORT
+  login:
+    home_redirect: null
+    branding: null
```
Comment (Amit-PivotalLabs) [inline spec/fixtures/aws/cf-stub.yml]: @Kaixiang Can you remove these 3 lines please?

**Comment by Kaixiang** [comment]
@Amit-PivotalLabs , see if this looks better?

**Pull request merged**